test/run_tests.js

```diff
@@ -80,6 +80,11 @@ const loadMocha = () => {
       // Skipping the file that manages shared state across tests.
       file => file !== path.join(__dirname, 'shared.js'),
     )
+    .filter(
+      file => file.indexOf(
+        'test/04_openst/gnosisSafeModules/delayedRecoveryModule/helpers/' === -1,
```

Comment:
I assume this is meant to filter out the helper files added in this PR, but I don't think it is filtering anything. Maybe the closing parenthesis on L86 is wrongly-placed?

Reply:
Yes. Wrongly placed. Fixed (thx!).